proto/xion/dkim/v1/state.proto

```diff
@@ -7,13 +7,13 @@ option go_package = "github.com/burnt-labs/xion/x/dkim/types";
 
 // https://github.com/cosmos/cosmos-sdk/blob/main/orm/README.md
 
-message DkimPubKeys {
+message DkimPubKey {
     option (cosmos.orm.v1.table) = {
       id: 1;
-      primary_key: { fields: "domain" }
-      index: { id: 1 fields: "pub_key" unique: true }
+      primary_key: { fields: "selector,domain" }
```

Comment:
How does the key get built with multiple fields? My guess is we probably want "domain,selector"

Reply:
I am not sure but I believe similar to a normal rdbms so combining the keys so I can swap this too